portal/templates/reporting_dashboard.html

```diff
@@ -6,6 +6,9 @@ <h3 class="tnth-headline">{{ _("Reporting Dashboard") }}</h3>
 
     <h2 class="tnth-headline">{{ _("Usage Statistics") }}</h2>
     <div>
+        <p>{{ _("Registrations are collected from the User.registration timestamps of all Users without the Test Role") }}</p>
+        <p>{{ _("Logins are collected from the start_time timestamps of Encounters whose auth_method='password_authenticated'") }}</p>
+        <p>{{ _("Intervention Logins are filtered based on the login Encounter's User's User.interventions") }}</p>
         <table id="usageStatsTable"
```

Comment:
@OptimusRhine can you please add that clarification?
so instead of this:
<p>{{ _("Intervention counts are tallied from User.interventions (e.g. a User with both the 'Care Plan' and 'Symptom Tracker' interventions, would add 1 to both Interventions' counts)") }}</p> 
this:
<p>{{ _("Intervention counts only apply to those interventions that control their subject list manually (eg Sexual Recovery, Care Plan, Community of Wellness). They are tallied from User.interventions (e.g. a User with both the 'Care Plan' and 'Community of Wellness' interventions, would add 1 to both Interventions' counts)") }}</p> 

In the next iteration we'll probably need to move P3P, Symptom Tracker and WiserCare to their own section.